Makefile

```diff
@@ -32,8 +32,6 @@ else ifneq (,$(findstring petting,$(example)))
 DOCKER_IMAGE_TAG=pz
 else ifneq (,$(findstring flatland,$(example)))
 DOCKER_IMAGE_TAG=flatland
-else ifneq (,$(findstring openspiel,$(example)))
```

Comment:
Think we should probably remove melting-pot as well, since all examples have been removed, it is no longer on README and we do not really support it atm

Reply:
@AsadJeewa  I think we will create another PR for it since it demands having many files to edit.

Reply:
We should just remove the OpenSpiel requirements from the setup.py.